pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -649,6 +653,13 @@ func (r *Reconciler) runReconcileSeedFlow(
 		)
 	}
 
+	var (
+		_ = g.Add(flow.Task{
+			Name: "Deploying cache Prometheus",
+			Fn:   cachePrometheus.Deploy,
+		})
```

Comment:
The CRDs are already applied here: https://github.com/gardener/gardener/blob/529b54fc5143ddd7824b1461eaf03c8b86daede3/pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go#L273-L275
Hence, we can deploy the custom resources even if the operator is not yet running (it will pick them up after start).